response_operations_ui/templates/new-enrolment-code.html

```diff
@@ -49,7 +49,7 @@ <h1 class="jupiter">New enrolment code</h1>
 
       <form action="/reporting-units/{{ru_ref}}">
         <input type="hidden" name="s" value="073" />
-        <button class="btn btn--primary">Ok</button>
+        <button class="btn btn--primary" id="ok_button">Ok</button>
```

Comment:
Might be worth using a dash to be consistent with `id` fields elsewhere?

Reply:
good point Liam, I named it using the standard now used in Frontstage but as this hasn't been converted yet, it needs a '-'